app/Transformers/UserStatisticsTransformer.php

```diff
@@ -49,6 +50,7 @@ public function transform(UserStatistics\Model $stats = null)
         ];
     }
 
+    // TODO: split this into global_rank (already exists) and country_rank
```

Comment:
it seems weird to want to split this but then have stats from multiple modes in a `statistics_all` field.